src/organization/organization.reducer.js

```diff
@@ -7,63 +19,65 @@ const initialState = {
   isPendingOrg: false,
   isPendingRepos: false,
   isPendingMembers: false,
-  error: '',
+  organizationError: '',
+  organizationRepositoriesError: '',
+  organizationMembersError: '',
 };
 
-export const organizationReducer = (state = initialState, action = {}) => {
-  switch (action.type) {
```

Comment:
OMG how have I not used `redux-actions` before, it looks so much cleaner :heart_eyes: